### PR TITLE
[ddp-server] Remove underscore

### PR DESCRIPTION
**Pull request by harryadel**

### CODE REVIEW

**Review by StorytellerCZ — APPROVED** [review]

**Comment by StorytellerCZ** [comment]
Merged from 2.14 as there have been some changes to the parts that the tests failed on (I think).

**Comment by StorytellerCZ** [comment]
Two failures:
```
S: tinytest - mongo-livedata - id-based invalidation, STRING failed: {"groupPath":["tinytest","mongo-livedata"],"test":"id-based invalidation, STRING","events":[{"sequence":5253,"type":"fail","details":{"type":"assert_equal","expected":"{\"all\":1,\"id2Direct\":1,\"id2InQuery\":1,\"bothIds\":1}","actual":"{\"all\":1,\"id1Direct\":1,\"id1InQuery\":1,\"id2Direct\":1,\"id2InQuery\":1,\"bothIds\":1}","not":false}}],"server":true}
S: tinytest - mongo-livedata - id-based invalidation, MONGO failed: {"groupPath":["tinytest","mongo-livedata"],"test":"id-based invalidation, MONGO","events":[{"sequence":6272,"type":"fail","details":{"type":"assert_equal","expected":"{\"all\":1,\"id2Direct\":1,\"id2InQuery\":1,\"bothIds\":1}","actual":"{\"all\":1,\"id1Direct\":1,\"id1InQuery\":1,\"id2Direct\":1,\"id2InQuery\":1,\"bothIds\":1}","not":false}}],"server":true}
```
I think these might be related to new changes.

**Comment by harryadel** [comment]
@StorytellerCZ :shipit:

**Review by StorytellerCZ — DISMISSED** [review]

**Comment by harryadel** [comment]
@denihs pls see if you can merge this one too in 3.0.3

File: packages/ddp-server/livedata_server.js [hunk 1]
```diff
@@ -639,7 +645,7 @@ Object.assign(Session.prototype, {
           return true;
         });
 
-        if (_.has(self.protocol_handlers, msg.msg)) {
+        if (has(self.protocol_handlers, msg.msg)) {
```
Comment (leonardoventurini) [inline packages/ddp-server/livedata_server.js]: Could this be something like `msg.msg in self.protocol_handlers`?

File: packages/ddp-server/crossbar.js [hunk 2]
```diff
@@ -88,13 +90,13 @@ _.extend(DDPServer._Crossbar.prototype, {
 
     var collection = self._collectionForMessage(notification);
 
-    if (! _.has(self.listenersByCollection, collection)) {
+    if (!has(self.listenersByCollection, collection)) {
```
Comment (leonardoventurini) [inline packages/ddp-server/crossbar.js]: Here too, can it use `in`?

**Review by leonardoventurini — APPROVED** [review]
Thanks @harryadel 

We're already releasing v3.0.3, so it can't go in this one, sorry

File: packages/ddp-server/livedata_server.js (hunk 1 above)
Comment (StorytellerCZ) [reply packages/ddp-server/livedata_server.js]: This seems to be less known syntax. Here is some documentation for folk: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

**Comment by denihs** [comment]
I've changed the base branch for this one. After you make the changes Leo mentioned, we can merge them and include them in the next release.

**Pull request merged**